src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -58,13 +58,22 @@ public async Task<(Error error, GitHubUser user)> GetByLogin(string login)
 
             var user = TryGetByLogin(login);
             if (user != null)
-                return user.IsValid() ? ((Error)null, user) : (Errors.GitHubUserNotFound(login), null);
+            {
+                user.Expiry = NextExpiry();
```

Comment:
or we can set the expiry per repo? 

---
In reply to: [223551363](https://github.com/dotnet/docfx/pull/3467#discussion_r223551363) [](ancestors = 223551363)

Reply:
This is per user, so that DocFX can recall GitHub to update cache if user changes its information on GitHub. So you are right that it should not be added here.

---
In reply to: [223551363](https://github.com/dotnet/docfx/pull/3467#discussion_r223551363) [](ancestors = 223551363)